virtual_rainforest/core/grid.py

```diff
@@ -64,7 +78,7 @@ def make_square_grid(
     cell_ny: int,
     xoff: float = 0,
```

Comment:
What is the offset for if we ask the user to preprocess all inputs?

Reply:
If a user has data from a real location with real coordinates in a projected coordinate system, then forcing them to change the origin of those coordinates is not helpful. And we want to create outputs that preserve the actual coordinates too, so that output GIS files can be plotted correctly in the real location.